scripts/globals/combat/magic_hit_rate.lua

```diff
@@ -17,14 +17,14 @@ local elementTable =
     [xi.magic.element.DARK   ] = { xi.mod.DARKACC,    xi.mod.DARK_AFFINITY_ACC,    xi.mod.DARK_MEVA,    xi.mod.DARK_RES_RANK,    0                                 },
 }
 
--- actor Magic Accuracy
-xi.combat.magicHitRate.calculateActorMagicAccuracy = function(actor, target, spell, skillType, spellElement, statUsed, bonusMacc)
-    local actorJob      = actor:getMainJob()
-    local actorWeather  = actor:getWeather()
-    local spellGroup    = spell and spell:getSpellGroup() or xi.magic.spellGroup.NONE
-    local statDiff      = actor:getStat(statUsed) - target:getStat(statUsed)
 
```

Comment:
CI not happy about the double newline here

Reply:
rebased and fixed. probably